types/leaflet/index.d.ts

```diff
@@ -1357,7 +1886,8 @@ export class GeoJSON<P = any> extends FeatureGroup<P> {
  * map (you can alternatively add it later with addData method) and
  * an options object.
  */
-export function geoJSON<P = any>(geojson?: geojson.GeoJsonObject, options?: GeoJSONOptions<P>): GeoJSON<P>;
```

Comment:
Add /** @deprecated */ on the one you do not like.

Reply:
AFAICT, neither is deprecated